packages/waku/src/router/client.ts

```diff
@@ -316,3 +319,34 @@ export function Router() {
     createElement(InnerRouter),
   );
 }
+
+function notAvailableInServer(name: string) {
+  return () => {
+    throw new Error(`${name} is not in the server`);
+  };
+}
+
+export function ServerRouter({
+  children,
+  loc,
+}: PropsWithChildren<{ loc: ReturnType<typeof parseLocation> }>) {
+  if (!isServer) {
+    throw new Error('ServerRouter is not supposed to be run in the client');
+  }
```

Comment:
We don't need this check (at least for now). But, maybe we need some comments telling it's not a public API, also for `ServerRoot`.